### PR TITLE
Added beer-song exercise solution

### PR DESCRIPTION
**Pull request by Russtopia**

### CODE REVIEW

File: exercises/practice/beer-song/.meta/config.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "blurb": "Recite the lyrics to that beloved classic, that field-trip favorite: 99 Bottles of Beer on the Wall.",
+  "authors": [
+    "FIXME-marnen"
```
Comment (Russtopia) [inline exercises/practice/beer-song/.meta/config.json:4]: whoops should must be `marnen` I suppose

File: config.json [hunk 2]
```diff
@@ -56,6 +56,18 @@
           "dates"
         ]
       },
+      {
+        "slug": "beer-song",
+        "name": "99 Bottles of Beer",
```
Comment (ErikSchierboom) [inline config.json:61]: I've checked, and all other tracks use `Beer Song` as the name, so it's probably best to be consistent:

```suggestion
        "name": "Beer Song",
```

File: exercises/practice/beer-song/.meta/config.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    ],
+    "test": [
+      "beer-song.tc"
+    ],
+    "example": [
+      ".meta/beer-song-example.apl"
+    ]
+  },
+  "source": "The traditional car-ride song, \"99 Bottles of Beer on the Wall\".",
+  "source_url": "https://en.wikipedia.org/wiki/99_Bottles_of_Beer"
```
Comment (ErikSchierboom) [inline exercises/practice/beer-song/.meta/config.json:21]:
```suggestion
  "source": "Learn to Program by Chris Pine",
  "source_url": "http://pine.fm/LearnToProgram/?Chapter=06"
```

See https://github.com/exercism/problem-specifications/blob/main/exercises/beer-song/metadata.toml

File: exercises/practice/beer-song/.meta/config.json (hunk 1 above)
Comment (ErikSchierboom) [reply exercises/practice/beer-song/.meta/config.json:4]: I'm slightly confused. Aren't you the author of this exercise? As in: haven't you been the one creating these files?

Comment (Russtopia) [reply exercises/practice/beer-song/.meta/config.json:4]: Yes. I was thinking for some reason this was the template's author. WIll change to myself thanks

Comment (ErikSchierboom) [reply exercises/practice/beer-song/.meta/config.json:4]: Side note: once done, you should remove yourself from the contributors list (you can be either an author _or_ a contributor).

File: exercises/practice/beer-song/.meta/config.json [hunk 4]
```diff
@@ -1,10 +1,10 @@
 {
   "blurb": "Recite the lyrics to that beloved classic, that field-trip favorite: 99 Bottles of Beer on the Wall.",
   "authors": [
-    "FIXME-marnen"
+    "russtopia"
   ],
   "contributors": [
-    "russtopia"
+    "ErikSchierboom"
```
Comment (ErikSchierboom) [inline exercises/practice/beer-song/.meta/config.json:7]: I've hardly done anything :)

```suggestion
```

A contributor should provide a substantial contribution, which I don't think I have :)

**Review by ErikSchierboom — APPROVED** [review]
One final nit.

File: exercises/practice/beer-song/.meta/config.json (hunk 4 above)
Comment (Russtopia) [reply exercises/practice/beer-song/.meta/config.json:7]: :) :+1:

**Comment by ErikSchierboom** [comment]
Merged! 🎉 

Side note: you might want to fix the lint errors first: https://github.com/exercism/gnu-apl/runs/4365004790?check_suite_focus=true That way, you'll notice any actual errors in PRs easier.

**Comment by marnen** [comment]
I thought I was still the primary maintainer for this track, so I’m confused as to how pull requests are getting merged without my input. (I’ll be the first to admit that I haven’t given this track the time it deserves, so I’m not sure this is a *bad* thing, but I feel like I’m a bit blindsided here.) How is the process supposed to work?

**Pull request merged**